static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -94,13 +92,25 @@ function GroupRelatedIssues({params}: Props) {
               <HeaderWrapper>
                 <Title>{t('Issues caused by the same root cause')}</Title>
                 {sameRootCauseIssues.length > 0 ? (
-                  <GroupList
-                    endpointPath={`/organizations/${orgSlug}/issues/`}
-                    orgSlug={orgSlug}
-                    queryParams={{query: `issue.id:[${sameRootCauseIssues}]`}}
-                    query=""
-                    source="related-issues-tab"
-                  />
+                  <div>
+                    <TextButtonWrapper>
+                      <div />
```

Comment:
Leaving this div to spread the LinkButton to the right.
<img width="790" alt="image" src="https://github.com/getsentry/sentry/assets/44410/158a9f15-054a-4d5c-b043-aeb2fa80731f">